.travis.yml

```diff
@@ -1,13 +1,55 @@
 language: rust
 sudo: false
 
+GH_TOKEN: &GH_TOKEN
+    secure: kkW+IvXcFzk91ozPv/dBVA3aZ3tb7QJNSXeqzUct56+TfK6uKFiaFim+lOz8D7mDAa1Jbr1qACHPwaUNmY/ksGGWK75llnh8rub2+GVRKs+TEqqZCFBGo3bXHSp7setxYJ1RMbP0bI1vwf1+7pZCSpog0Qyx7JJfayxRO8lEfI9lJSgWA6Hxk2fMdj+614zRN2GbtXBEBU16msk2rYxmnX3sgSPg+TMK7uiSZVOO0fqvunGy2nbqyzqrU58QrsQ9B6f1a4qpCrTkuO3fZgqqU1U9jGFGemOp84qAkVoNc+4s+NM9KydwKyChW0/nnD88325xJ0jVGB7eglFQLedRygNQ1qVsdzCdqFWi2Ui2jHGYd+dMC73cG/GpU2fYtwbz3lZwO/rz2XpLyWOfHoVNZT+mRD3srSsDXbZhOf1/uRxdDx+ra719bBTy2BAGS6wEKrldiWM/jKZU2I4JKyY26cfaoDnYhbBq3qNpXppnceGsDlbrt3EA/7uUVPhidq18NEAICAgdkhMTVBvDIlEj8mYvAWAt9cSmDoDNnx+4kPcIVCz1HL/YryV5QHbfuowVKvRnZe8dH1g/KiEODli3xxrakzfbTwRYOJTjPzh7JCHS//P1aIKd51sJ3XkmMvI47RMSKgvOBTsSVKkksjdjOemv+K0o48492QClRv+nrqE=
+
+DEPLOY_TO_GITHUB: &DEPLOY_TO_GITHUB
+  before_deploy:
+    - git config --local user.name "Ashley Williams"
+    - git config --local user.email "ashley666ashley@gmail.com"
```

Comment:
What operation follows that uses git? the deploy thing will use the encrypted token afaik

Reply:
i dunno i'm just following what i have from wasm-pack...